src/main/java/com/gachigang/ontherun/common/ApplicationConstants.java

```diff
@@ -28,5 +28,7 @@ public class Validation {
     public class Security {
         public static final String TOKEN_PREFIX = "Bearer ";
         public static final String AUTH = "auth";
+        public static final String[] ALLOWEDORIGINS = {"https://ontherun.com"};
+        public static final String[] ALLOWEDMETHODS = {"GET","POST"};
```

Comment:
FYI: If you name constant in Java (static final) it should be named LIKE_THIS, simply split words with undescore. Because in your approach it is hard to recognize difference

Reply:
I removed these two lines of code